src/Route.php

```diff
@@ -110,6 +125,11 @@ public function constructUrl(Request $appRequest, Nette\Http\Url $refUrl): ?stri
 
 	private function isHttpMethodSupported(string $httpMethod): bool
 	{
+		if ($this->allowOnceAllMethods) {
```

Comment:
It is quite confusing for me, it took me while to understand what this condition block does, for more clarity it would be better to tweak naming, decouple it to method or add comment to code.